pkg/arvo/app/link-view.hoon

```diff
@@ -0,0 +1,303 @@
+::  link-view: frontend endpoints
+::
+::  endpoints, mapping onto link-store's paths. p is for page as in pagination.
+::  updates only work for page 0.
+::  as with link-store, urls are expected to use +wood encoding.
+::
+::    /json/[p]/submissions                           pages for all groups
+::    /json/[p]/submissions/[some-group]              page for one group
+::    /json/[p]/discussions/[wood-url]/[some-group]   page for url in group
+::
+/+  *link, *server, default-agent, verb
+::
+|%
++$  state-0
+  $:  %0
+      ~
+  ==
+::
++$  card  card:agent:gall
+--
+::
+=|  state-0
+=*  state  -
+::
+%+  verb  |
+^-  agent:gall
+=<
+  |_  =bowl:gall
+  +*  this  .
+      do    ~(. +> bowl)
+      def   ~(. (default-agent this %|) bowl)
+  ::
+  ++  on-init
```

Comment:
This doesn't [launch the tile](https://github.com/urbit/urbit/blob/os1-rc/pkg/arvo/app/link-server-hook.hoon#L98).